ding/policy/ppg.py

```diff
@@ -792,30 +899,37 @@ def _forward_learn(self, data: dict) -> Dict[str, Any]:
         Overview:
             Forward and backward function of learn mode.
         Arguments:
-            - data (:obj:`Dict[str, Any]`): Dict type data, a batch of data for training, values are torch.Tensor or \
-                np.ndarray or dict/list combinations.
+            - data (:obj:`Dict[str, Any]`): Input data used for policy forward, including the \
+                collected training samples from replay buffer. For each element in dict, the key of the \
+                dict is the name of data items and the value is the corresponding data. Usually, the value is \
```

Comment:
"Usually, the value is ..." ->
"Usually, the class type of value is either torch.Tensor or np.ndarray, or a dict/list containing either torch.Tensor or np.ndarray items"